.github/release_pull_request_template.md

```diff
@@ -0,0 +1,42 @@
+This is the release pull request for {version} of the WooCommerce Blocks plugin.
+
+## Communication
+
+<!--
+  This section is for any notes related to communicating the release.
+  Please include any extra details with each item as needed.
+-->
+
+* [ ] What is being introduced in this release?
+* [ ]  If this release has potentially **high customer impact**, is a blog post (section) ready? Provide a link if applicable.
+* [ ] Is the **relevant developer documentation updated**? Please provide links if applicable.
+* [ ] Any special instructions or helpful notes for Happiness engineers are posted (this might not be needed for every release).
```

Comment:
Just a small comment, we could redo this section to show the structure of the communication test we're after rather than having bullets. It might be easier to write that way and avoid checklist items for things not applicable.

e.g. 

```
{version} introduces the following major features...

Developer documentation for this release covers:

- {subject}
```

Just a thought.

Reply:
Ya I like that idea, I'll work out something for that.